src/components/Books.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+// import { v4 as uuidv4 } from 'uuid';
+// import { addBook } from '../redux/books/books';
+import BookList from './BookList';
 import AddBooks from './AddBook';
 
-const Books = () => (
-  <AddBooks />
-);
+const selectBooks = (state) => state.books;
+
+const Books = () => {
+  // const [title, setTitle] = useState('');
+  // const [author, setAuthor] = useState('');
+  // const dispatch = useDispatch();
```

Comment:
[Optional] Kindly remove disabled Linters, Linting improves readability, removes errors before execution, and also make your code more professional ✊

Please check for other occurrences. 🙂